pandas/tests/indexes/common.py

```diff
@@ -127,6 +127,13 @@ def test_ndarray_compat_properties(self):
 
         values = idx.values
         for prop in self._compat_props:
+            # skip equivalency check if converted type is object, as happens
+            # for PeriodIndex, since then object (address) would be 4 bytes
+            # on 32bit platforms and equivalence to int64 of original
+            # date time is just accidental
+            if prop in ('itemsize', 'nbytes') \
+                    and values.dtype.name == 'object':
+                continue
```

Comment:
@jreback - I have added one more "skip" which was failing on i386 -- described "incode".  Does it make sense? ;)